configuration/globalproperties/activePatients.xml

```diff
@@ -0,0 +1,34 @@
+<config>
```

Comment:
@prashantsable1991 please delete the file `activePatientsByLocation.xml` that is updating the same global property